server/sonar-web/src/main/js/apps/quality-profiles/details/ProfilePermissions.tsx

```diff
@@ -0,0 +1,153 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import { uniqBy } from 'lodash';
+import ProfilePermissionsUser from './ProfilePermissionsUser';
+import ProfilePermissionsAddUserForm from './ProfilePermissionsAddUserForm';
+import { searchUsers } from '../../../api/quality-profiles';
+import { translate } from '../../../helpers/l10n';
+
+export interface User {
```

Comment:
Would it be possible to reuse the type from `api/quality-profiles.ts` ? (you might have to change a bit the definition since it's an array)